string-manipulation-2/src/swap-chars.js

```diff
@@ -0,0 +1,20 @@
+/* exported swapChars */
+// create storage for the word with swapped characters in the form of an array
+// iterate over each letter of the string
+//  if the current letter is equal to the first index provided:
+//    set the letter at the first index equal to the letter at the second given index
+//    set the letter at the second index equal to the letter at the first index
+// return the allocated storage which includes the swapped letters and turn it back into a string
+
+function swapChars(firstIndex, secondIndex, string) {
+  var newString = string.split('');
+  var indexOne = newString[firstIndex];
+  var indexTwo = newString[secondIndex];
+  for (var i = 0; i < string.length; i++) {
+    if (string[i] === string[firstIndex]) {
```

Comment:
This loop and conditional isn't necessary, you can just do the work directly after you've converted the string to an array